packages/maps/src/Client.ts

```diff
@@ -17,7 +17,8 @@ export class Maps {
    * @param options
    */
   public async instantiateMapFrom(options: MapOptions) {
-    const { sql, dataset, vector_extent = 2048, vector_simplify_extent = 2048, metadata = {}, aggregation = {} } = options;
+    const { sql, dataset, vector_extent = 2048, vector_simplify_extent = 2048,
```

Comment:
We're going to add a common formatter with prettier, so we all get homogeneous spacing automagically